Docs/Configuration.tex

```diff
@@ -3532,15 +3562,15 @@ \subsection{Security Properties}\label{miscsecurityprops}
   \href{https://github.com/acidanthera/bugtracker/issues/995}{acidanthera/bugtracker\#995}
   for more details.
 
-  \emph{Note 2}: Resetting NVRAM will also erase all the boot options
-  otherwise not backed up with bless (e.g. Linux).
+  \emph{Note 2}: Resetting NVRAM will also erase any boot options, such as
+  Linux boot options, not backed up using the bless command.
```

Comment:
`which are not backed up` I guess?

Reply:
This is wrong and gets even more wrong. Linux options generally follow bless model and thus do not die. Yet some users may create custom boot options, usually for Linux. These do die. In my opinion, revert.